source/Xunit.Gherkin.Quick.UnitTests/ScenarioOutlineExecutorTests.cs

```diff
@@ -49,6 +49,36 @@ public async Task ExecuteScenarioOutlineAsync_Requires_FeatureInstance()
         private sealed class FeatureForNullArgumentTests : Feature
         { }
 
+		[Fact]
+		public async Task ScenarioOutline_Runs_Background_Steps_First()
+		{
+			var feature = new GherkinFeatureBuilder()
+				.WithBackground(sb => sb
+					.Given("background", null))
+				.WithScenarioOutline("test outline", sb => sb
+					.Given("I chose <a> as first number", null)
+					.And("I chose <b> as second number", null)
+					.When("I press add", null)
+					.Then("the result should be <sum> on the screen", null),
+					 eb => eb
+						.WithExampleHeadings("a", "b", "sum")
+						.WithExamples("", db => db.WithData("1", "2", "3")))
+				.Build();
+			
+			_featureFileRepository.Setup(r => r.GetByFilePath($"{nameof(FeatureWithScenarioSteps)}.feature"))
+				.Returns(new FeatureFile(new Gherkin.Ast.GherkinDocument(feature, new Gherkin.Ast.Comment[0])))
+				.Verifiable();
```

Comment:
Same here - please verify that this setup did run in the assertions section, by calling `Verify()`. You can refer to one of the existing unit tests for example if needed.

Reply:
Done